src/ServiceProvider.php

```diff
@@ -2,15 +2,17 @@
 
```

Comment:
I'd like to keep this file as it was before, with the name `CanvasServiceProvider`